### PR TITLE
add ASF DOAP file (Description of a Project)

### PR DESCRIPTION
**Pull request by johnmccabe**
- https://projects.apache.org/project.html?brooklyn
- https://projects-old.apache.org/doap.html

@rdowner @ahgittin @aledsage - can you review please, I suspect the long description needs overhaul (pretty much a carbon copy from the website)

I suggest we use the following URL to specify the location of the DOAP file, gives us flexibility when updating.

- https://git1-us-west.apache.org/repos/asf?p=brooklyn.git;a=blob_plain;f=doap_Brooklyn.rdf;hb=HEAD

### CODE REVIEW

File: doap_Brooklyn.rdf [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    <repository>
+      <GitRepository>
+        <location rdf:resource="https://git-wip-us.apache.org/repos/asf/brooklyn.git"/>
+        <browse rdf:resource="https://git-wip-us.apache.org/repos/asf?p=brooklyn.git"/>
+      </GitRepository>
+    </repository>
+    <maintainer>
+      <foaf:Person>
+        <foaf:name>Richard Downer</foaf:name>
+          <foaf:mbox rdf:resource="mailto:richard.downer@cloudsoftcorp.com "/>
```
Comment (rdowner) [inline doap_Brooklyn.rdf]: Please use "richard@apache.org" - thanks!

Can we specify multiple maintainers? If so then please add all the Brooklyn committers:

```
    Aled Sage                  <aledsage@apache.org>            [2015-11-18]
    Alex Heneveld              <heneveld@apache.org>            [2015-11-18]
    Andrea Turli               <andreaturli@apache.org>         [2015-11-18]
    Andrew Kennedy             <grkvlt@apache.org>              [2015-11-18]
    Ciprian Ciubotariu         <cipi@apache.org>                [2015-11-18]
    Hadrian Zbarcea            <hadrian@apache.org>             [2015-11-18]
    Jean-Baptiste Onofré       <jbonofre@apache.org>            [2015-11-18]
    Olivier Lamy               <olamy@apache.org>               [2015-11-18]
    Richard Downer (chair)     <richard@apache.org>             [2015-11-18]
    Sam Corbett                <sjcorbett@apache.org>           [2015-11-18]
    Svetoslav Neykov           <svet@apache.org>                [2015-11-18]
```

File: doap_Brooklyn.rdf [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+        Multi-tiered applications described in Blueprints can be deployed, configured and integrated across 20+ public
+        clouds (Amazon EC2, CloudStack, OpenStack, SoftLayer and many more), private clouds, Docker containers or bare
+        servers.
+
+        Once deployed Apache Brooklyn monitors key application metrics to provide dynamic policy management, including
+        autoscaling to meet demand, and resilisancy though replacing/restarting failed components.
+
+        Management of Apache Brooklyn is made possible via a dynamic web console, rich REST API and simple yet powerful
+        command line interface.</description>
```
Comment (rdowner) [inline doap_Brooklyn.rdf]: Looking at a few other projects on Apache, it seems that this is usually just a single paragraph. How about this:

"Brooklyn is about deploying and managing applications: composing a full stack for an application; deploying to cloud and non-cloud targets; using monitoring tools to collect key health/performance metrics; responding to situations such as a failing node; and adding or removing capacity to match demand."

I think that'll be good enough to get us going, and we can iterate on it over time.

(lifted from https://brooklyn.apache.org/learnmore/theory.html)

**Comment by rdowner** [comment]
Thanks @johnmccabe this is great! Only a couple of comments then it can be merged. Then you can tell me how I install it on apache.org :stuck_out_tongue_winking_eye:

**Comment by johnmccabe** [comment]
@rdowner you (or another ASF committer) needs to add the URL pointing at the DOAP to this file (we can just pull from the apache git repo using the link in my first comment - looks like others are doing the same):

- http://svn.apache.org/viewvc/comdev/projects.apache.org/data/projects.xml?view=markup

The site is updated from this at approx 2am UTC I believe.

**Comment by rdowner** [comment]
Perfect @johnmccabe - I'll get this merged and linked from apache.org. Thanks for sorting this out :smile:

**Comment by rdowner** [comment]
It's merged and I've updated the data file for projects.apache.org. All going well it should be live by tomorrow morning...

Thanks again for taking care of this, it's been on my TODO list for a while...

**Comment by Humbedooh** [comment]
testing webhooks, ignore me :)

**Pull request merged**